src/components/AddPlaidBankAccount.js

```diff
@@ -163,13 +186,22 @@ class AddPlaidBankAccount extends React.Component {
                                     label={this.props.translate('addPersonalBankAccountPage.chooseAccountLabel')}
                                     onChange={(index) => {
                                         this.setState({selectedIndex: Number(index)});
+                                        const errors = this.getErrors();
+                                        if (!errors.selectedBank) {
+                                            return;
+                                        }
+
+                                        const newErrors = {...errors};
```

Comment:
Is the destructuring necessary here? Why can't we just delete the `selectedBank` property from `errors` and pass that? Same question for the BankAccountStep.js addition.

Reply:
This SO post explain a bit about why https://stackoverflow.com/questions/37358135/can-i-delete-props-in-react

But the basic idea is that you want your props to be fed from whatever state management you're using (in this case Onyx) and should avoid mutating them (errors is a reference to the errors object in props). So we're making a copy and then deleting the property from there and then setting the errors again.

Reply:
Ah ok makes sense 👍